packages/runtime/src/next/components/tests/controls/color-control/server.test.tsx

```diff
@@ -0,0 +1,32 @@
+import { Color, ColorData } from '@makeswift/controls'
+import { testPageControlPropRendering } from '../page-control-prop-rendering'
+import { swatch, value, cacheData } from './fixtures'
+
+describe.each([
+  {
+    version: 0,
+    toData: (value: ColorData) => value,
+  },
+  {
+    version: 1,
+  },
+])('color data v$version', ({ toData }) => {
+  test.each([
+    {
+      value,
+      swatch,
+      defaultValue: 'rgb(255,0,0)',
+    },
+    {
+      value,
+      swatch: null,
+      defaultValue: 'rgb(255,0,0)',
+    },
+  ])(`reading swatch $swatch from cache`, async ({ value, defaultValue, swatch }) => {
```

Comment:
Only testing reading the swatch from cache (we prefetch all resources before rendering on the server).